pkgs/tools/security/b2sum/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, openmp }:
+
+stdenv.mkDerivation rec {
+  version = "20160619";
+  name = "b2sum-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/BLAKE2/BLAKE2/archive/${version}.tar.gz";
+    sha256 = "0csnlp6kwlyla5s4r6bsrx2jgcwrm9qzisnvfdhmqsz5r8y87b6b";
+  };
+  postUnpack = "sourceRoot=$sourceRoot/b2sum";
```

Comment:
`sourceRoot = "b2sum";`?

Reply:
Um, I’m not sure it would work this way, because the source root is `<whatever the dir in the archive is called>/b2sum`